<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FITBIT</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/window.png</normaloff>:/images/window.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
border-image:url(:/images/bg1.jpg)

}</string>
   </property>
   <widget class="QWidget" name="logo" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#logo{
	border-image: url(:/images/IMG_1668.png);
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>291</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{
	border-image: url(:/images/Untitled_design.png);
}</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>241</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nu">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#nu{
color:white;
}</string>
        </property>
        <property name="text">
         <string>NewUser? SignUp</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label1">
        <property name="font">
         <font>
          <family>Bahnschrift SemiCondensed</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
color:white;
}</string>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="username_2"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label2">
        <property name="font">
         <font>
          <family>Bahnschrift SemiCondensed</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
color:white;
}</string>
        </property>
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="password_2">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label3">
        <property name="font">
         <font>
          <family>Bahnschrift SemiCondensed</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
color:white;
}</string>
        </property>
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="email_2">
        <property name="styleSheet">
         <string notr="true">#lineEdit{
color:white;
}</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhEmailCharactersOnly</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label4">
        <property name="font">
         <font>
          <family>Bahnschrift SemiCondensed</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{

color:white;
}</string>
        </property>
        <property name="text">
         <string>Phone:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLineEdit" name="phone_2"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="register_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#register_2 {
    
	background-color: rgb(255, 215, 0);
	color: rgb(0, 0, 0);
 border-radius:15px solid black;
	}

	QPushButton#register_2:hover{
    background-color: rgb(0, 142, 213);
	}

QPushButton#register_2:pressed{
    background-color: rgb(0, 26, 141);
	}</string>
     </property>
     <property name="text">
      <string>REGISTER</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="text">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>640</y>
      <width>122</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiCondensed</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#text{

color: rgb(255, 255, 255)
}</string>
    </property>
    <property name="text">
     <string>Already a User :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>640</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiCondensed</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton{
background-color: rgb(255, 215, 0);
color: rgb(0, 0, 0);
border-radius: 10px;
border-style: solid;
}

QPushButton#pushButton:hover{
    background-color: rgb(0, 142, 213);
	}

QPushButton#pushButton:pressed{
    background-color: rgb(0, 26, 141);
	}</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
